### PR TITLE
Add isAttachmentElement

### PR DESCRIPTION
**Pull request by grayswan**

### CODE REVIEW

**Review by chrisworkmode — APPROVED** [review]

**Comment by chrisworkmode** [comment]
Does this need to be built and have the dist committed too?

**Comment by jahndis** [comment]
> Does this need to be built and have the dist committed too?

I'll take care of that as part of the release process.

**Review by jahndis — APPROVED** [review]
I'll go ahead and take care of the version bumping and releasing.

**Review by chrisgregorysni — APPROVED** [review]

File: src/form-field-schema.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ export default class FormFieldSchema {
         this.addElementColumn(element, 'country');
       }
 
-      if (this.fullSchema && (element.isPhotoElement || element.isVideoElement || element.isAudioElement)) {
+      if (this.fullSchema && (element.isPhotoElement || element.isVideoElement || element.isAudioElement || element.isAttachmentElement)) {
```
Comment (clewis) [inline src/form-field-schema.js:70]: The next if, in operator.js, has element.isSignatureElement as well.  It seems odd that it's not here.

Comment (jahndis) [reply src/form-field-schema.js:70]: it looks like isSignatureElement is handled separately on line 75? Is that what you mean?

Comment (clewis) [reply src/form-field-schema.js:70]: Yeah, that works.  I don't understand what this is doing, just looking for inconsistencies.

**Pull request merged**